cfe_internal/enterprise/mission_portal.cf

```diff
@@ -74,6 +74,17 @@ classes:
           "max_spare_servers":"${php_fpm_www_pool_max_spare_servers}"
          }');
 
+  files:
```

Comment:
Here the cfe_enterprise_disable_http_redirect_to_https class guards management of the new flag file that PHP uses to know if redirection should be enabled or disabled.